django/contrib/staticfiles/storage.py

```diff
@@ -69,7 +69,8 @@ class HashedFilesMixin:
             "*.js",
             (
                 (
-                    r"(?m)(?P<matched>)^(//# (?-i:sourceMappingURL)=(?P<url>.*))$",
+                    r"(?m)(?P<matched>)"
+                    r"^(//# (?-i:sourceMappingURL)=(?P<url>.*?))[ \t]*$",
```

Comment:
I'm not sure why :thinking: but the new test works for me even without `\t` in the regexp.

Reply:
Yes, IMO it's still worth adding tests.